.github/actions/spelling/allow.txt

```diff
@@ -1,122 +1,51 @@
-abstract
```

Comment:
I've removed words that are in the base dictionary as there's no reason to keep them in a supplemental dictionary.